libraries/AP_HAL_SITL/SITL_State.h

```diff
@@ -106,23 +106,23 @@ class HALSITL::SITL_State {
     void _gps_write(const uint8_t *p, uint16_t size, uint8_t instance = 0);
```

Comment:
I think we should remove the default value here, this way it forces all new drivers to support both GPS instances.